src/Components/Organisms/carousel/carousel.tsx

```diff
@@ -0,0 +1,73 @@
+import "../carousel/carousel.scss";
+import CarouselContent from "../../molecules/carousel-content/carousel-content";
+import CarouselPreviousButton from "../../atoms/carousel-previous-button/carousel-previous-button";
+import CarouselNextButton from "../../atoms/carousel-next-button/carousel-next-button";
+import { CarouselButtons, CarouselData } from "../../../data";
+import jsonFile from "../../../db.json";
+import { useState } from "react";
+
+interface CarouselProps {
+  carouselData: CarouselData;
+  carouselButtonData: CarouselButtons;
+}
+const Carousel = (props: CarouselProps) => {
+  const carousel = jsonFile.carousel;
+
+  const [slideIndex, setSlideIndex] = useState(1);
+  const [slideDirection, setSlideDirection] = useState("right");
+  const [prevSlide, setPrevSlide] = useState(carousel.length);
+  const [blockedAnimation, setBlockedAnimation] = useState(true);
+
+  const nextSlide = () => {
```

Comment:
optional, loadNextSlide could be more appropriate name
nextSlide sounds like a an object describing the next slide
same thing for previousSlide